짝수와-홀수.js

```diff
@@ -0,0 +1,5 @@
+function solution(num) {
+    var answer = "";
+    num % 2 === 0 ? (answer = "Even") : (answer = "Odd");
```

Comment:
삼항연산자 안에 answer를 대입하는 것 보다 대입을 앞에 쓰는게 더 간결할 거 같아요
그리고 0이랑 1은 자동으로 불리언 값으로 변환되니까 생략할 수 있어요
```suggestion
    answer = (num % 2) ? "Odd" : "Even"
```